CONTRIBUTING.adoc

```diff
@@ -0,0 +1,240 @@
+[[contributing-to-omni-core]]
+= Contributing to Omni Core
+DexX7
+:jbake-type: page
+:jbake-status: published
+:jbake-tags: omni, specification, whitepaper
+:idprefix:
+:numbered:
+:toc: macro
+:toclevels: 1
+
+toc::[]
+
+// AsciiDoc supports comments that are not rendered.
+
+NOTE: We're currently using "NOTE" sections for TODOs.
+
+NOTE: The numbered lists with keyword-ish descriptions are mostly notes, and
+should completely be refined and replaced.  It may not the best choice to simply have some lists, but instead it
+would probably better to write full texts. If you have good ideas, or want to tackle it, please just add inline
+comments!
+
+NOTE: Some introduction should go here. What??
+
+
+[[reporting-security-issues]]
+== Reporting Security Issues
+
+Issues with potential security implications should not be published, but
+instead communicated via email to [... where?].
+
+This includes especially defects and potential exploits, and other
+pitfalls that may break consensus of the Omni Layer.
+
+Please don't hesitate, if you are unsure, whether your observation
+qualifies as security related issue.
+
+NOTE: It should quickly be described what happens after a report was filed. It should be mentioned that reports can be send encrypted (list keys).
+It should probably be mentioned that we confirm the receipt of a
+report, but likely won't confirm, whether it was a false alarm or not. +
+We should mention that findings are going to be disclosed after they
+were fixed, and the contributor might be notified in such a case, if desired?
+
+[[reporting-other-issues]]
+== Reporting Other Issues
+
+[[contributors]]
+=== Contributors
+
+1.  contributors should try to describe the observed behavior of the
+potential issue +
```

Comment:
@msgilligan: are the ` +` needed here? At least in the preview of [Atom](https://atom.io/) via [asciidoc-preview](https://github.com/asciidoctor/atom-asciidoc-preview) it doesn't seem to make a difference.